src/ArrayOfPrimeNumber.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Created by Yura on 02.05.2019.
+ */
+import java.util.Arrays;
+
+public class ArrayOfPrimeNumber {
+    public static void main(String[] args) {
+        int a = 100;
```

Comment:
Nice try, but this number (and numbers alike) should be extracted as field like
private static final int MORE_MEANINGFUL_NAME = 100;